src/webapi/Service/PeriodicHostedSyncService.cs

```diff
@@ -0,0 +1,67 @@
+namespace eppeta.webapi.Service
+{
+    class PeriodicHostedSyncService : BackgroundService
+    {
+        private readonly TimeSpan _period = TimeSpan.FromDays(1);
+        private readonly ILogger<PeriodicHostedSyncService> _logger;
+        private readonly IServiceScopeFactory _factory;
+        private int _executionCount = 0;
+        public bool IsEnabled { get; set; } = true;
+
+        public PeriodicHostedSyncService(
+            ILogger<PeriodicHostedSyncService> logger,
+            IServiceScopeFactory factory)
+        {
+            _logger = logger;
+            _factory = factory;
+        }
+
+        protected override async Task ExecuteAsync(CancellationToken stoppingToken)
+        {
+            // ExecuteAsync is executed once and we have to take care of a mechanism ourselves that is kept during operation.
+            // To do this, we can use a Periodic Timer, which, unlike other timers, does not block resources.
+            // But instead, WaitForNextTickAsync provides a mechanism that blocks a task and can thus be used in a While loop.
+            using PeriodicTimer timer = new PeriodicTimer(_period);
+
+            // When ASP.NET Core is intentionally shut down, the background service receives information
+            // via the stopping token that it has been canceled.
+            // We check the cancellation to avoid blocking the application shutdown.
+            while (
+                !stoppingToken.IsCancellationRequested &&
+                await timer.WaitForNextTickAsync(stoppingToken))
+            {
+                try
```

Comment:
@DavidJGapCR I don't think we shouldn't have try/catches, I'm pretty sure the logging middleware expects exceptions to be thrown. See /infrastructure/LoggingMiddleware.cs for details